db/migrate/20151018090641_create_persistent_volume_claim.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePersistentVolumeClaim < ActiveRecord::Migration
+  def change
+    create_table :persistent_volume_claims do |t|
+      t.belongs_to :ems, :type => :bigint
+      t.belongs_to :persistent_volume, :type => :bigint
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :ems_created_on
+      t.timestamp  :created_on
+      t.string     :resource_version
+      t.text       :desired_access_modes, :array => true, :default => []
+      t.string     :volume_name
+      t.string     :phase
+      t.text       :actual_access_modes, :array => true, :default => []
+      t.text       :capacity
```

Comment:
@blomquisg I think volume_name and phase might be the same but not the capacity field, @simon3z you agree?